spec/02-integration/04-core/02-hooks_spec.lua

```diff
@@ -7,13 +7,29 @@ local pl_path = require "pl.path"
 local pl_file = require "pl.file"
 local pl_stringx = require "pl.stringx"
 
+local api_client
+
+-- cache entry inserted as a sentinel whenever a db lookup returns nothing
+local db_miss_sentinel = { null = true } 
+
+local function get_cache(key)
+  local r = assert(api_client:send {
+    method = "GET",
+    path = "/cache/"..key,
+    headers = {}
+  })
+  assert.response(r).has.status(200)
+  return assert.response(r).has.jsonbody()
+end
+
 describe("Core Hooks", function()
   describe("Global", function()
-    describe("Global Plugin entity invalidation on API", function()
-      local client, api_client
+    describe("Plugin entity invalidation on API", function()
+      local client
       local plugin
 
       before_each(function()
+        helpers.dao:truncate_tables()
```

Comment:
@thibaultcha I had to insert the above truncation of the tables, or the tests would continue to give me an error that an entry already existed for `hooks2.com` below.

Did recently anything change here, as I was under the impression that the tables would automatically truncate upon starting/stopping kong in tests.

More odd; it only gave me the error `hooks2.com`, but not on `hooks1.com`.

any ideas?